### PR TITLE
MSA: Fix template (max_safe_path_cost)

### PR DESCRIPTION
**Pull request by mintar**
Also fix a typo in a debug message.

### CODE REVIEW

File: moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch [hunk 1]
```diff
@@ -16,7 +16,7 @@
   <arg name="allow_trajectory_execution" default="true"/>
   <arg name="fake_execution" default="false"/>
   <arg name="execution_type" default="interpolate"/> <!-- set to 'last point' to skip intermediate trajectory in fake execution -->
-  <arg name="max_safe_path_cost" default="1"/>
+  <arg name="sense_for_plan/max_safe_path_cost" default="1"/>
```
Comment (rhaschke) [inline moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch:19]: Indeed, the dynamic reconfigure server for this parameter is placed within the namespace `sense_for_plan`:
https://github.com/ros-planning/moveit/blob/abb83f178075b4d5bb655df2dc9dd3e400e258ca/moveit_ros/planning/plan_execution/src/plan_with_sensing.cpp#L52-L53

**Review by rhaschke — APPROVED** [review]
Thanks for this fix. 
Please, provide an explanation / motivation for such a configuration change in future. This will simplify our review process.

**Review by rhaschke — CHANGES_REQUESTED** [review]
Actually, you fixed the name in the wrong location. You need to adapt the `param` name here:
https://github.com/ros-planning/moveit/blob/781a82c19a0d9acc7b22a2bcbab0ec0f7e5e27c4/moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch#L91

instead of the `arg` name.

**Comment by mintar** [comment]
Ooops, good catch. I copy-pasted the stuff from my own moveit config to the template and messed up. It's fixed now.

**Review by rhaschke — APPROVED** [review]

**Comment by v4hn** [comment]
Thanks. PlanWithSensing is definitely not something many people use with MoveIt these days. :cat2:

**Comment by mintar** [comment]
> Thanks. PlanWithSensing is definitely not something many people use with MoveIt these days.

I'm not using it either, I just noticed this while debugging #2702.

**Pull request merged**